src/App.js

```diff
@@ -1,12 +1,17 @@
 import "./App.css";
 import Nav from "./components/Nav";
 import Home from "./pages/home/Home";
-
+import { Route, Routes } from "react-router-dom";
+import Product from "./pages/product_listing/product";
```

Comment:
Suggestion: You can put a `index.js` file inside pages folder and export all the pages from there.
Then you don't need to write this long import line for each page:
Currently for Home and Product, you are doing:
```javascript
import Home from "./pages/home/home"
import Product from "./pages/product_listing/product"
```
After doing the suggestions you just have to do:
```javascript
import {  Home, Product } from "./pages"
```